docs/docs/stories.mdx

```diff
@@ -18,23 +18,23 @@ Here's an example of a dialogue in the Rasa story format:
 stories:
 - story: collect restaurant booking info  # name of the story - just for debugging
   steps:
-  - intent: greet                         # user message with no entities
-  - action: utter_ask_howcanhelp
-  - intent: inform                        # user message with no entities
+  - intent: greet                          # user message with no entities
+  - action: utter_ask_howcanhelp           # action that the bot should execute
+  - intent: inform                         # user message with entities
     entities:
     - location: "rome"
     - price: "cheap"
-  - action: utter_on_it                  # action that the bot should execute
+  - bot: On it                             # actual text that bot can output
   - action: utter_ask_cuisine
-  - intent: inform
-    entities:
-    - cuisine: "spanish"
+  - user: I would like [spanish](cuisine). # actual text that user input
   - action: utter_ask_num_people
 ```
 
 ### User Messages
 
-While writing stories, you do not have to deal with the specific contents of
+While writing stories, you have two options to represent user input.
+
+1) You do not have to deal with the specific contents of
```

Comment:
Is this actually rendering as proper MD list?

Reply:
update so it is the same as other lists